AzureDevOps.Authentication/Src/MsaAuthentication.cs

```diff
@@ -33,8 +33,7 @@ namespace AzureDevOps.Authentication
 
     public sealed class MsaAuthentication : Authentication, IMsaAuthentication
     {
-        public const string DefaultAuthorityHost = AzureDevOps.Authentication.Authority.AuthorityHostUrlBase + "/live.com";
```

Comment:
nit: I see that `AzureDevOps.Authentication.Authority.DefaultAuthorityHostUrl` is already defined with the required "common" url. You can consider using that directly